api/nakadi-event-bus-api.yaml

```diff
@@ -1947,16 +1947,16 @@ definitions:
   CursorDistanceResult:
     allOf:
       - $ref: '#/definitions/CursorDistanceQuery'
-    required:
-      - distance
-    properties:
-      distance:
-        type: number
-        format: int64
-        description: |
-          Number of events between two offsets. Initial offset is exclusive. It's only zero when both provided offsets
-          are equal.
-
+      - type: Object        
+        properties:
+          distance:
+            type: number
+            format: int64
+            description: |
+              Number of events between two offsets. Initial offset is exclusive. It's only zero when both provided offsets
+              are equal.
+        required:
+          - distance
```

Comment:
what is the purpose of these changes? For me the original formatting looks pretty good and swagger-editor doesn't complain as well.